src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -181,8 +131,8 @@ export const ProfilePage = () => {
                       ? ownerAssets.map((asset, index) => (
                           <div key={index}>
                             <ViewMyNftsAsset
+                              isSellMode={false}
```

Comment:
setting this to true would actually allow listing to be reachable in between yours and @lynnshaoyu 's PR

Reply:
genius